stern/stern.go

```diff
@@ -94,10 +94,11 @@ func Run(ctx context.Context, config *Config) error {
 		containerStates:        config.ContainerStates,
 	})
 	newTail := func(t *Target) *Tail {
+		sinceSeconds := int64(config.Since.Seconds())
 		return NewTail(client.CoreV1(), t.Node, t.Namespace, t.Pod, t.Container, config.Template, config.Out, config.ErrOut, &TailOptions{
 			Timestamps:   config.Timestamps,
 			Location:     config.Location,
-			SinceSeconds: int64(config.Since.Seconds()),
+			SinceSeconds: &sinceSeconds,
```

Comment:
I prefer to use `k8s.io/utils/pointer` here.

```suggestion
			SinceSeconds: pointer.Int64(int64(config.Since.Seconds())),
```

Reply:
I fixed it in https://github.com/stern/stern/pull/230/commits/51f9032d342b07eb080592275dbf52f5e3865ae7 👍